### PR TITLE
Fix actions endpoint for block dropdowns

### PR DESCRIPTION
**Pull request by maths22**
Make it so you can use this library for interactive dropdowns.  Those callbacks don't have a response url, so make it optional.

### CODE REVIEW

**Comment by dblock** [comment]
Thanks, care to put this in CHANGELOG please, and maybe write a quick test with an actual payload?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 #### 0.3.2 (Next)
```
Comment (dblock) [inline CHANGELOG.md:4]: Put back this line.

**Review by dblock — CHANGES_REQUESTED** [review]
Still needs tests.

**Comment by dblock** [comment]
Want to finish this @maths22 ?

**Comment by maths22** [comment]
Whoops, I forgot about this; will add tests

**Comment by maths22** [comment]
@dblock done and confirmed it passes locally; travis doesn't seem to be running to validate it there

**Comment by dblock** [comment]
It's cause this:

Screen Shot 2022-02-28 at 6 07 38 PM

We'll have to migrate this project to GHA, maybe someone (you?) could take care of it.

**Comment by CrazyOptimist** [comment]
@maths22 you can merge `master` into your branch then CI will pass.

**Comment by maths22** [comment]
Ok it should be ready to run tests once you approve running them

**Comment by CrazyOptimist** [comment]
@dblock `danger` is failing with this message:
```
Missing ENV['DATABASE_ADAPTER'].
Could not find the type of CI for Danger to run on.
```
Have you experienced the same? Please gimme some hint if yes.  
I will investigate it and commit on the next PR(mongo v5 one).

**Comment by dblock** [comment]
I merged this, care to PR an update to that dangerfile @CrazyOptimist and let's see if that happens with the change?

**Comment by CrazyOptimist** [comment]
Roger!

**Pull request merged**